lib/ultimate_turbo_modal/base.rb

```diff
@@ -33,9 +36,11 @@ def initialize(
     @footer_divider = footer_divider
     @header = header
     @header_divider = header_divider
+    @title_tag = title_tag
```

Comment:
Why not reuse what we already have (ie `title_block`)? Also, shouldn't modify the base class.